msteams-platform/concepts/device-capabilities/browser-device-permissions.md

```diff
@@ -9,26 +9,38 @@ ms.topic: how-to
 
-### Personal apps and task module dialogs
-The "app permissions" setting can be found in the top-right.
+## Enable app permissions in the browser
+If your Teams app has declared in the [application manifest](native-device-permissions.md#specify-permissions) that it needs device permissions, then the **App permissions** appears for the users to enable the app's device permissions. The **App permissions** option is available in the following capabilities: 
+
+* Personal apps and task module dialogs
+* Chats, channel, or meeting tabs
+
+Personal apps and task module dialogs, the **App permissions** option is in the upper-right corner of the page.
+
+</br>
 <img src="../../assets/images/tabs/apppermissions.png" alt="App permissions button" width="800"/>
 
-### Chat, channel or meeting tabs
-The "app permissions" setting can be found in the tab dropdown.
+Chat, channel, or meeting tabs, the **App permissions** option is in the dropdown of the tab.
+
+
 ![App permissions drop-down](../../assets/images/tabs/drop-downapppermissions.png)
 
-A user will need to enable these permissions in the browser for these permissions to take effect. Once a user changes an app’s device permissions in the browser, they will be prompted to reload the application in Teams. It is important that you make users aware of where to go in order to enable these permissions in Microsoft Teams.
+After the **App permissions** option is selected a popup displays, where the user can enable the permissions button.
+
+A user will need to enable these permissions in the browser for these permissions to take effect. After user changes the app’s device permissions in the browser, they're prompted to reload the application in Teams.
+
+> [!IMPORTANT]
+> Ensure that the users are aware on how to enable these **App permissions** in Microsoft Teams.
 
 ## Recommendation
-Microsoft Teams applications that require device permissions in the browser are expected to show instructions to users on where to find and enable these permissions in the Teams UI. Depending on the context in which your application is running, you will need to ensure your instructions are pointing the user to correct location to access these permissions as they differ for personal apps, task module dialogs, and tabs in chats, channels or meetings.
+Teams app that require device permissions in the browser must show instructions to users where to find and enable these permissions in the Teams UI. Depending on the context in which your application is running, you need to ensure your instructions are pointing the user to correct location to access these permissions, as they differ for personal apps, task module dialogs, tabs in chats, and channels or meetings.
```

Comment:
Please change to:

Teams app that require device permissions in the browser must show instructions to users on where to find and enable these permissions in the Teams UI. Depending on the context in which your application is running, you need to ensure that your instructions are pointing the user to correct location to access these permissions, as they differ for personal apps, task module dialogs, tabs in chats, and channels or meetings.